pkg/controller/controller_test.go

```diff
@@ -7046,3 +7048,22 @@ func createFakeCSIPV(volumeHandle string) *v1.PersistentVolume {
 		},
 	}
 }
+
+type ProtobufMatcher struct {
+	// Protobuf messages implement String() function that renders the message as "protobuf style string".
+	// `name:"test-testi"  capacity_range:{required_bytes:100000}  parameters:{key:"fstype"  value:"ext3"}  parameters:{key:"translated"  value:"foo"}`
+	msg proto.Message
+}
+
```

Comment:
nit: Since we are defining a new matcher, canonical way of defining this in the context of mock library appears to be defining a function which returns matcher interface. So something like - https://github.com/uber-go/mock/pull/63/files

Reply:
I can handle this nit when I move the matcher over to csi-test, if we wanted to get this PR in for next provisioner release.

Reply:
I added `Protobuf()` function to return the matcher. Better name is welcome. And indeed, this belongs to csi-test.